qa/os/src/test/java/org/elasticsearch/packaging/test/ArchiveTests.java

```diff
@@ -311,7 +311,7 @@ public void test73CustomJvmOptionsDirectoryFilesWithoutOptionsExtensionIgnored()
             startElasticsearch();
 
             final String nodesResponse = makeRequest(Request.Get("http://localhost:9200/_nodes"));
-            assertThat(nodesResponse, containsString("\"heap_init_in_bytes\":1073741824"));
+            assertThat(nodesResponse, not(containsString("\"heap_init_in_bytes\":536870912")));
```

Comment:
This changes makes me uncomfortable. Is it not possible that we might happen to calculate the heap to be this size, causing the test to fail?

Reply:
I've rewritten the test to pass a bogus option, so that java should fail to start if the option is used.